### DISCUSSION TITLE
Please review and merge or close our open pull requests

### DISCUSSION THREAD

**Discussion by cclauss** [TheAlgorithms/Python#8583] [Team Posts]
We have about 200 open pull requests.  https://github.com/TheAlgorithms/Python/pulls

As a Python maintainer, please try to take some time to review some of these (especially those submitted this month) and merge or close them based on the guidelines in [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).  If you merge any pull requests, please add the [hacktoberfest-accepted](https://github.com/TheAlgorithms/Python/pulls?q=is%3Apr+is%3Aopen+label%3Ahacktoberfest-accepted) label.

**Comment by dhruvmanila** [comment]
FYI, the label is not required. The repository already contains the `hacktoberfest` topic which is enough for all PRs to be accepted for the event.

**Comment by putul03** [comment]
Hi @cclauss @dhruvmanila and the team! I understand you have a lot on your plate with many open pull requests. If you could take a moment to review my PR - Added Python Implementation of Suffix Arrays and LCP Arrays #12171, it would be greatly appreciated. I'm excited to contribute to the project and welcome any feedback you may have. Thank you!
